website/docs/minitest.md

```diff
@@ -187,3 +187,415 @@ def self.test_each_hash(hash, &blk)
   hash.each(&blk)
 end
 ```
+
+## RSpec
+
+Sorbet supports RSpec's testing DSL using the same spec method translation as Minitest. RSpec and Minitest share common constructs like `describe`, `it`, `before`, `after`, and `let`, which are all translated to Ruby classes and methods before type checking.
+
+### Using RSpec.describe
+
+To enable proper type checking with RSpec's custom matchers (like `eq`, `include`, `match_array`, etc.), use `RSpec.describe` instead of bare `describe` at the top level:
+
+```ruby
+RSpec.describe MyClass do
+  it 'works' do
+    expect(MyClass.foo).to eq 'foo'
+  end
+
+  context 'nested contexts' do
+    it 'works' do
+      expect(MyClass.foo).to eq 'foo'
+    end
+  end
+
+  describe 'nested describes' do
+    it 'works' do
+      expect(MyClass.foo).to eq 'foo'
+    end
+  end
+end
+```
+
+The `RSpec.` prefix ensures `RSpec::Core::ExampleGroup` is part of the ancestry chain of the translated classes.
```

Comment:
I can see the justification for having some docs about why the prefix is necessary, but I think it should be more along the lines of "The prefix is necessary for Sorbet to correctly identify RSpec matchers" or something like that.